tests/integration/multi_cluster_deploy_test.go

```diff
@@ -28,8 +29,7 @@ type MultiRookClusterDeploySuite struct {
 
 //Deploy Multiple Rook clusters
 func (mrc *MultiRookClusterDeploySuite) SetupSuite() {
-	//Skipping test until https://github.com/rook/rook/issues/970 is fixed.
-	mrc.Suite.T().SkipNow()
```

Comment:
how much time does this test add?

Reply:
about 4 minutes, which I'm OK with since it explicitly validates that Rook supports multi-tenancy, both clusters are healthy, and that file, block and object storage is functional.  I really like the addition of this test suite.